waypoints_joints:
  home:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  sleep:
  - -0.0015339808305725455
  - -1.7426022291183472
  - -1.5063692331314087
  - -0.8007379770278931 
  grab:
  - 0.18867963552474976
  - 0.23623304069042206
  - -0.058291271328926086
  - -1.225650668144226
